trade-finance-manager-ui/templates/case/amendments/amendment-answers.njk

```diff
@@ -79,6 +79,21 @@
       }
     %}
     {{- optionsWithHTML.push(defaultCoverEndDate) | reject() }}
+
+    {% if isTfmFacilityEndDateFeatureFlagEnabled %}
+      {% set defaultIsUsingFacilityEndDate =
+        {
+          key:{ text: 'Bank provided a facility end date' },
+          value: { html: '<span data-cy="amendment--is-using-facility-end-date-response">' + ( "Yes" if isUsingFacilityEndDate === true else "No") + '</span>' },
+          actions: { items: [ {
+            html: '<span data-cy="amendment--is-using-facility-end-date-link">Change</span>',
+            href: '/case/' + dealId + '/facility/' + facilityId + '/amendment/' + amendmentId + '/is-using-facility-end-date',
+            visuallyHiddenText: 'new is using facility end date'
```

Comment:
Have you checked this with the designers?
```suggestion
            visuallyHiddenText: 'new is using facility end date'
```

Reply:
Ooh good spot, that's a typo from me. Based on the pattern of other rows in that file, it seems like the `visuallyHiddenText` should be the same as the visual text, so will amend it to `bank provided a facility end date`. If you still think I should check let me know!

Reply:
Sounds good :)